README.rst

```diff
@@ -166,6 +166,8 @@ Below is the help output::
                             in a mess
       --range line line     apply docformatter to docstrings between these lines;
                             line numbers are indexed at 1
+      --docstring-length length length
```

Comment:
"length" seems duplicate

Reply:
It is a range, like `line line` just above.  That could be made clearer for sure.

Reply:
I've just blindly duplicated the doc for `--range`, what would be clearer?

Reply:
Having different names for the params, for example `min_length max_length`

Reply:
I've updated the PR with your wording.